### PR TITLE
Added references to validator status page

### PR DESCRIPTION
**Pull request by nv-harvey**
Included in the deployment guide and the monitoring page

### CODE REVIEW

File: docs/mine-hnt/validators/mainnet/deployment-guide.mdx [hunk 1]
```diff
@@ -476,4 +476,6 @@ The `swarm_key` equates to your validator's unique identity on the Helium blockc
 
 Head over to the [Validator Mainnet Explorer](https://explorer-beta.helium.com/validators) and search for your Validator. This may take 10 to 15 minutes from the time that your Validator is both online and staked before it appears on explorer. You should see your 3-word validator name (`short-umber-bull` in the example above) listed. It may be easier to sort by `#`, descending, as your new Validator will be near the top of the list. Remember you can view the name of your validator through `miner info summary` or just `miner info name`.
 
+Visit the [Helium Validator Status Page](http://validators-status.helium.com/) (and sign up for alerts, if you haven't already). This status page will provide the latest updates on important issues that impact validators (chain halts or other major events).
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/mainnet/deployment-guide.mdx]:
```suggestion
Visit the [Helium Validator Status Page](http://validators-status.helium.com) (and sign up for alerts, if you haven't already). This status page will provide the latest updates on important issues that impact validators like chain halts or other major events.
```

File: docs/mine-hnt/validators/monitoring.mdx [hunk 2]
```diff
@@ -9,6 +9,12 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 The Docker commands will assume you have the same prefix to get you executing a command within the docker: ```docker exec validator``` . If you want to make it easier, you can always create an alias such as: ```alias miner="docker exec validator miner"```.
 
+## Confirm the network is working
+
+Visit the [Helium Validator Status Page](http://validators-status.helium.com/) (and sign up for alerts, if you haven't already).
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/monitoring.mdx]:
```suggestion
Visit the [Helium Validator Status Page](http://validators-status.helium.com) (and sign up for alerts, if you haven't already).
```

File: docs/mine-hnt/validators/monitoring.mdx [hunk 3]
```diff
@@ -9,6 +9,12 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
...
+
+This status page will provide the latest updates on important issues that impact validators (chain halts or other major events).
```
Comment (PharkMillups) [inline docs/mine-hnt/validators/monitoring.mdx]:
```suggestion
This status page will provide the latest updates on important issues that impact validators like chain halts or other major events.
```

**Review by PharkMillups — COMMENTED** [review]
Thanks! Left a few small suggestions. Take a look at those then we can get it merged.

**Comment by nv-harvey** [comment]
Thanks @PharkMillups . All those changes looked good to me. I think this now incorporates them (I'm not the most fluent github user..!)

**Review by cokes518 — APPROVED** [review]

**Pull request merged**